R/get_uid.R

```diff
@@ -75,13 +75,34 @@
 #' }
 
 get_uid <- function(sciname, ask = TRUE, verbose = TRUE, rows = NA){
+  repeat_until_it_works <- function(func, catch, max_tries = 3, wait_time = 10, verbose = TRUE, ...) {
```

Comment:
Yea, thats a good idea. If we use `...` then we should probably make `repeat_until_it_works` an independent exported function so that we can refer to its documentation in `get_uid`'s documentation. It might be useful in other functions anyway.